README.md

```diff
@@ -374,6 +380,55 @@ You can save up to 90% of cost when you use Spot Instances. You just need to mak
 
 :warning: **Note**: If the price of the instance reaches this point it will be terminated.
 
+### Worker Group Launch Templates
+
+Worker Groups, the default worker node groups for this module, are based on an older AWS tool called "Launch Configurations" which have some limitations around Spot instances and delegating a percentage of a pool of workers to on-demand or spot instances, as well as issues when autoscaling is enabled.
+
+The issue with autoscaling with the default worker group is that it is prone to autoscaling using Nodes from only a single AZ.
+AWS has a "AZRebalance" job that can run to help with this, but it is aggressive in removing nodes.
+
+All of these issues can be resolved by using Worker Group Launch Templates instead, configured with a template for each Availability Zone. 
+Using an ASG for each AZ bypasses the autoscaling issues in AWS.
+Furthermore, we are also able to specify several types of machines that are suitable for spot instances rather than just one.
+Using only one often results in Spot instances not being able to be provisioned, and this greatly reduces the occurence of this happening, as well as allowing for lower spot prices.
+
+#### Enabling Worker Group Launch Templates
+
+To use the Worker Group Launch Template, set the variable `enable_worker_groups_launch_template` to `true`, and define an array of instance types allowed.
+
+When using autoscaling with Launch Templates per AZ, the min and max number of nodes is per zone.
+These values can be adjusted by using the variables `lt_desired_nodes_per_subnet`, `lt_min_nodes_per_subnet`, and `lt_max_nodes_per_subnet`
+
+```terraform
```

Comment:
Move this from the markdown to the examples. I am working on a PR to add tests, so planning to just test everything in the examples folder.